### PR TITLE
Issue #2322 :doing 'comit' after 'move'

### LINKED ISSUE #2322: Doing `commit` after `move`
Hi, after using `dvc move` to move versioned data files, when doing `dvc status` one gets `changed checksum` so these files need to be `dvc commit`. I suggest combining `move+commit` because this is obviously what we want to do.

### PR DESCRIPTION
**Pull request by kss682**
Fix : stage.save() added at#L68

Fixes #2322 

* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

**Comment by shcheklein** [comment]
@kss682 we should def add a test for this

File: dvc/repo/move.py [hunk 1]
```diff
@@ -65,5 +65,5 @@ def move(self, from_path, to_path):
 
     with self.state:
         out.move(to_out)
-
+    stage.save()
```
Comment (efiop) [inline dvc/repo/move.py]:
```suggestion
        stage.save()
```

**Comment by kurianbenoy** [comment]
@kss682  don't forget to complete the pending work :smile:

**Comment by efiop** [comment]
@kss682 Looks like something went wrong during rebase/merge, eh? 🙂Looks like you didn't add any tests yet, so if you want to, you could close this PR and create a new one instead 🙂

**Comment by kss682** [comment]
@efiop Ok I will close this PR .

**Pull request closed without merging**